tests/unit/panther_analysis_tool/test_lookup_tables.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
Do you want this empty file?

Reply:
It's a placeholder, I still need to add tests specifically for lookup tables. Need to get the existing custom schema tests working again first.